hypothesis-python/src/hypothesis/extra/numpy.py

```diff
@@ -1312,7 +1312,10 @@ def do_draw(self, data):
         else:
             while result[-1:] == [slice(None, None)] and data.draw(st.integers(0, 7)):
                 result.pop()
-        return tuple(result)
+        if len(result) == 1 and data.draw(st.booleans()):
+            return result[0]
+        else:
+            return tuple(result)
```

Comment:
```suggestion
        if len(result) == 1 and data.draw(st.booleans()):
            # Sometimes generate bare element equivalent to a length-one tuple
            return result[0]
        return tuple(result)
```

This is a slightly more compact way to write the same logic, and a comment to remind my future self what we're doing here!

Reply:
Ok, I just found out about the _Changelog Entries_ in the `documentation.rst`. I've to admit I've never used very much ReStructuredText.